src/main/resources/org/cbioportal/persistence/mybatisclickhouse/StudyViewFilterMapper.xml

```diff
@@ -105,6 +105,121 @@
+                                <if test="dataFilterValue.value == 'MUTATED'">
+                                    SELECT sample_unique_id FROM mutated_samples
+                                </if>
+                                <if test="dataFilterValue.value == 'NOT_MUTATED'">
+                                    SELECT sample_unique_id FROM profiled_samples
+                                    WHERE sample_unique_id NOT IN (SELECT sample_unique_id FROM mutated_samples)
+                                </if>
+                                <if test="dataFilterValue.value == 'NOT_PROFILED'">
+                                    SELECT sample_unique_id FROM total_samples
+                                    WHERE sample_unique_id NOT IN (SELECT sample_unique_id FROM profiled_samples)
+                                </if>
+                            </foreach>
+                        )
+                    </if>
+                    <!-- if the categorization is 'MUTATION_TYPE' (table) -->
+                    <if test="mutationDataFilter.categorization == @org.cbioportal.web.parameter.MutationOption@MUTATION_TYPE">
+                        <foreach item="dataFilterValues" collection="mutationDataFilter.values" separator="INTERSECT">
+                            SELECT DISTINCT sample_unique_id
+                            FROM genomic_event_derived
+                            WHERE hugo_gene_symbol = #{mutationDataFilter.hugoGeneSymbol}
+                                AND variant_type = 'mutation'
+                                AND mutation_type IN (
+                                <foreach item="dataFilterValue" collection="dataFilterValues" separator=",">
+                                    #{dataFilterValue.value}
+                                </foreach>
+                                )
+                        </foreach>
+                    </if>
+                </foreach>
+            </if>
+            <if test="studyViewFilter.genomicDataFilters != null and !studyViewFilter.genomicDataFilters.isEmpty()">
+                <foreach item="genomicDataFilter" collection="studyViewFilter.genomicDataFilters" open="INTERSECT" separator="INTERSECT">
+                    <bind name="valueColumn" value="genomicDataFilter.profileType + '_value'" />
+                    <!-- 1. check if 'NA' is selected -->
+                    <bind name="containsNA" value="false" />
+                    <foreach item="dataFilterValue" collection="genomicDataFilter.values">
+                        <if test="dataFilterValue.value == 'NA'">
+                            <bind name="containsNA" value="true" />
+                        </if>
+                    </foreach>
+                    WITH cna_query AS (
+                        SELECT sample_unique_id, ${valueColumn}
+                        FROM genetic_alteration_derived_cna
```

Comment:
lets discuss this name.  i think derived should always be at the end of the table name.  i do like the idea that the name contains the underlying table from which it's derived.  
generic_alteration_cna_derived
genetic_alteration_generic_assay_derived ?
lets discuss